docs/reference/pathformat.rst

```diff
@@ -112,21 +112,28 @@ will expand to "[2008]" for one album and "[2010]" for the other. The
 function detects that you have two albums with the same artist and title but
 that they have different release years.
 
-For full flexibility, the ``%aunique`` function takes two arguments, each of
-which are whitespace-separated lists of album field names: a set of
-*identifiers* and a set of *disambiguators*. Any group of albums with identical
-values for all the identifiers will be considered "duplicates". Then, the
-function tries each disambiguator field, looking for one that distinguishes each
-of the duplicate albums from each other. The first such field is used as the
-result for ``%aunique``. If no field suffices, an arbitrary number is used to
-distinguish the two albums.
+For full flexibility, the ``%aunique`` function takes three arguments. The
+first two are whitespace-separated lists of album field names: a set of
+*identifiers* and a set of *disambiguators*. The third argument is a pair of
+characters used to surround the disambiguator.
+
+Any group of albums with identical values for all the identifiers will be
+considered "duplicates". Then, the function tries each disambiguator field,
+looking for one that distinguishes each of the duplicate albums from each
+other. The first such field is used as the result for ``%aunique``. If no field
+suffices, an arbitrary number is used to distinguish the two albums.
 
 The default identifiers are ``albumartist album`` and the default disambiguators
 are ``albumtype year label catalognum albumdisambig``. So you can get reasonable
 disambiguation behavior if you just use ``%aunique{}`` with no parameters in
 your path forms (as in the default path formats), but you can customize the
 disambiguation if, for example, you include the year by default in path formats.
 
+The default characters used as brackets are ``[]``. To change this, provide a
+third argument to the ``%aunique`` function consisting of two characters: the left
+and right brackets. Or, to turn off bracketing entirely, use a single blank
+space.
```

Comment:
Right—I meant that, to get no brackets, you would use `%aunique{foo,bar,}` instead of `%aunique{foo,bar, }`. To use the default square brackets, you would leave the comma off: `%aunique{foo,bar}`. In other words, the third argument to the function would be `''` for no brackets and `None` for the default.